components/count-badge/README.md

```diff
@@ -25,6 +25,17 @@ The `d2l-count-badge` element is a web component to display a number count, eith
 | `max-digits`, default: `2` when `type="notification"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and no limit for `"count"` type.
 | `hide-zero`, default: `false` | Boolean | Optionally choose not to show the count badge when the number is zero. |
 
+### Accessibility Properties
+
+To make your `d2l-count-badge` accessible, use the following properties when applicable:
+
+| Attribute | Description |
+|--|--|
+| `description` | Optionally choose to add a description, to be the aria-label on the badge. |

```

Comment:
Wondering: without this, a screen reader user is just going to happen upon a random number when combing through the page. Should we rename this to `label` and make it required?

Reply:
In its current state, without the tab stop, the label just never gets read. There are some cases, like card-footer-link, where screen readers don't read the count badge at all, so no label is required.

Reply:
What about just arrowing through? It should get read then. Like most non-interactive things on the page (headings, paragraphs, lists, etc) don't get tab stops either -- the screen reader user navigates to them using other means.

Reply:
I see what you mean now. Then it does make sense to make it required, I'll send an update with the *hopefully* fixed tests.